### ISSUE TITLE
MEmu Thread.. perhaps the last stand of it

### ISSUE THREAD

**Issue by AkagiShiroe** [wDCat/ANRC#98]
HI! we want to say.. its not modified.. and wasn't possible to use hack with this. please consider this as.. nothing 🤣

**Comment by RinFallen** [comment]
Oh lol :v Memu is dead again and Remix OS Dual Boot still alive.

**Comment by AkagiShiroe** [comment]
perfect right? :v found a way to repack .OVA let see if i can reconstruct this sh*t based on cat "detected files information".

**Comment by reihunter360** [comment]
You can thank the guide makers out there for fucking around.

**Comment by AkagiShiroe** [comment]
@reihunter360 everythings is good until that "massive reroll" happening, yeah thx a lot xD

**Comment by TonyJaa212** [comment]
Ohhh noooo

**Comment by darkzamurai** [comment]
She messed up no working root bypass for FGO.....

On Jul 31, 2017 10:21 PM, "TonyJaa212" <notifications@github.com> wrote:

Ohhh noooo

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/wDCat/ANRC/issues/98#issuecomment-319081603>, or mute
the thread

**Comment by cubanodun** [comment]
at this point the only option is to crack the apk like perfare did back in the day, this is getting ridiculous, they are reaching Denuvo levels of madness now 

@Teodor19 if you find someone that crack the apk, try to not make it "very" public, every time all their anti emulators methods come straight from the reddit

**Comment by chernobylzambie** [comment]
Hopefully a crack or new method is found, i can't get the current methods to work so far between bluescreens on vbox and invisible files on remix os and fgo is too strong for my current devices. I don't have a couple hundred spare dollars lying around for anything strong enough to play it... :c hopefully some kind (or incredibly bitter) soul is willing to work on this.

Thank you everyone for your hard work so far, it really helps

**Comment by ScienceOfMemory** [comment]
The only way a reliable and permanent fix is being made is if it isn't shared with reddit because they're too ignorant of the real world to not pointlessly share it around like it is free donuts.

**Comment by darkzamurai** [comment]
I know there's mod apk exist but you buy first VIP in androidrepublic they mod the apk that can bypass root with hacks US/NA and JPN..... When you get their apk and try to know how did they do it.....

**Comment by cubanodun** [comment]
@darkzamurai the problem is not root this time, is that the apk is scanning for specific core emulator files, ofc i also think that the method must be the same that the anti root check

someone has to bite the bullet and share with a few people the apk i would do if it not were for my shitty country not letting me buy anything in dollars

**Comment by ScienceOfMemory** [comment]
The android republic apk does not actually work on Memu because of the license file it requires to function. Further, it doesn't actually bypass the ant-emu security, so even if the license worked, it still would not work on Memu. And then STILL, the apk you buy, or rather the license you buy, has an activation limit of 2 devices, so sharing is largely pointless. But feel free to waste your money.

**Comment by darkzamurai** [comment]
Uh ok..... Its only for androie phones and tablets right?.... Not emu?

**Comment by ScienceOfMemory** [comment]
It works on some emus.
Technically.
But while the License works, it does not bypass emu security, so it just closes on boot. So phones (and maybe tablets. I don't know about tablets) only.

**Comment by AkagiShiroe** [comment]
any hope using RemixOS on vbox or vmware?.. just wanna login.. that anniversary reward is too much for losing.

**Comment by ScienceOfMemory** [comment]
Just use dual-boot then

**Comment by AkagiShiroe** [comment]
dual-boot doesn't support Dial-up for unrooted one, i don't have Wi-fi or ethernet in my place

**Comment by AkagiShiroe** [comment]
hey @wDCat (and teodore but that guy doesn't here).. any worth checking LineageOS? there's a version that was working for PC ([HERE](http://geektillithertz.com/wordpress/index.php/2017/01/03/lineageos-13-android-x86-on-pc/)] it could be work for dualboot or even emulated using vbox and vmware since someone has done what you done cat, [[HERE](https://www.reddit.com/r/grandorder/comments/6kcuih/guide_to_play_fgo_on_unrooted_lineageos/?ref=search_posts)], could be usefull sample to unroot since LineageOS still can play FGO (both JP and US).

**Comment by Teodor19** [comment]
I'm everywhere. I'll see. Though even if it'll work, it won't change the fact of 3fps gameplay on VBOX.
Though what i currently investigate is double emulation win->linux->android. And so far i'm stuck at launching emu in linux.

**Comment by AkagiShiroe** [comment]
@Teodor19 better 3 fps than nothing for someone who doesn't have phone yet. goodluck on that.

**Comment by AkagiShiroe** [comment]
found a way to remove(hidden in theory since the data are still written but without header any application shouldn't find it) memu files, hey @wDCat which file fgo searching for? are your last posted screenshoot is all of them?..

**Comment by reihunter360** [comment]
-snip

**Comment by AkagiShiroe** [comment]
@reihunter360 allready found that chinese-name XyDownloader.apk(not detected anymore on system), and also i replaced default launcher with other (apk and backup is deleted completly..).
the only problem are im not experienced with Android enviroment (specially on they program code) which mean i dunno which file are getting flagged.. or i still get missing a few file?. (still got error 60).

**Comment by reihunter360** [comment]
It was MemuLauncher.Apk that made allowed it to run.

**Comment by AkagiShiroe** [comment]
@reihunter360 u mean the launcher wasn't replaced? i all ready check it.. MEmulauncher are wiped clean without leaving trace and i set other launcher as default one. but it could be pain in ass if i tried to hide whole system file.. i have to track each file hex one by one 🤣

**Comment by reihunter360** [comment]
MemuLauncher.APK was the one that made FGO run. It had a emulation hider you see.

**Comment by AkagiShiroe** [comment]
@reihunter360 i though it was Poketest from cat who make memu can run fgo xD not the launcher.

**Comment by Teodor19** [comment]
It's probably just thing that catches mail with error info. Without need to be logged in in some other mail app.

Nothing on my side. Notebook died a little bit, so i'm using my other PC. Problem is, it's AMD and seen by Phoenix/Remix/Lineage as 32bit for some reason. Phoenix still shows 60, but i still haven't edited default.prop, because Phoenix doesn't want to start up after i edit ramdisk.
And of course i haven't logged in FGO for days and lost 10 tickets, bleh.

**Comment by AkagiShiroe** [comment]
@Teodor19 as long it was virtual OS (or you could say it using VMDK/IMG Base example like RemixOS) i could manipulate data header and make the file gone for good while system can read it for booting. welp im not trying to modifying memu yet. busy with my life here~.

**Comment by cubanodun** [comment]
Did not AMIDuOS use vbox?

**Comment by AkagiShiroe** [comment]
![screenshot_2017-08-09-07-10-59](https://user-images.githubusercontent.com/14356240/29115664-5de88c84-7d33-11e7-9cb2-5f985b7a13c4.png)
new shit from DW?

**Comment by Teodor19** [comment]
What did you use?

**Comment by AkagiShiroe** [comment]
@Teodor19 my old memu with fgo 1.25, update it to 1.27. they seperate it from root detection to emu?

**Comment by Teodor19** [comment]
Well, emu was 10, but it could be separation, yes. They probably want to decrease workload even further.

**Comment by AkagiShiroe** [comment]
@Teodor19 now what i need is probably a leak of it.. probably which file get flagged.. still figuring out but its hard to hide all of it except one by one.
(hope @wDCat are ressurected.. at least bring some info on this 70)

**Comment by Teodor19** [comment]
1.27.1 is out.
32bit version of ROS and POS do not work, even though FGO only has arm and x86 libs. I guess i really have to wait for my notebook to be fixed.

**Comment by AkagiShiroe** [comment]
error 70 seems get fixed by 1.27.1 but still back with error 60.. need some method to fix this.. hate to admit someone does can play with "fixed.ova" but they won't share it.. (RinFallen inform me)

**Comment by cubanodun** [comment]
@AkagiShiroe at least that means there is a way, for some reason i cant use Remix OS in dual boot because Remix refuse to show video, no matter how many times i try it just show a black screen and do nothing 

Also in the case someone here manage to crack it, can we decided on a controlle method of distribution? im tired already of delight works fucking with legit emu users just because some chineses want to make a quick buck

**Comment by Teodor19** [comment]
Well, emus/vms didn't actually die even once, so there's always a way if person have money or knowledge. Not sharing saves time for making new workaround after  some aniplex/sw guy decides to walk through baidu/reddit/whateverplace.

FGO ROS has a lot of allergies. Try using kagoromo's method to rule out any incompatibilities with my or someone else's system file.
EDIT: Well, i can keep it to just myself if i'm told to do so in exchange for not dualboot method. If i (haha) find something, i'll probably still distribute it freely. (if it's not related to ROS, as it seems to be safe-bay for some reason)

**Comment by AkagiShiroe** [comment]
@cubanodun @Teodor19 about sharing method.. i'll probably using xdelta patch for patchin system.img only since all backup use same one (if this work.. will save lot bandwith and we can try to create new VM to reduce size), welp still working on it. i forgot to change the keyboard :lol that's the root of problem.

**Comment by cubanodun** [comment]
@AkagiShiroe sounds fine to me, @Teodor19 is the emu guy in reddit so give it to him so he can share with people he see fit (dont forget about me wink wink) dont want edgelords and rerollers ruining the fun

**Comment by AkagiShiroe** [comment]
@cubanodun done reversing all of it ( i mean deleting most file getting flagged) the problem are IMEI and DEVICE Info.. probably if that show unknown.. fgo will automatically flagging 60 code. have to rewrite the code.. wish me luck~

**Comment by AkagiShiroe** [comment]
note: i see hope.. fully reversed (with other launcher and swift).. game are running about 1 sec (black screen) the goes off get flagged (error 60 code) same problem with VMWare.. welp, i need to reverse the IMEI so it can show fake-information like that hardenedloader.

**Comment by AkagiShiroe** [comment]
found it.. MEmuIME was keyboard (im stupid :lol ).. welp seem device are connected with MEmuSF and MEmuguest.. have to rework it

**Comment by AkagiShiroe** [comment]
done fully reversed memu.. but as long this problem doesn't fixed.. it won't launch fgo i guest?.. or im missing something.. here a bit screenshot:
![1](https://user-images.githubusercontent.com/14356240/29137619-e74d1600-7d7b-11e7-8011-8e5b140b0d04.jpg)
![2](https://user-images.githubusercontent.com/14356240/29137617-e735072c-7d7b-11e7-83e0-99115104a436.jpg)
![3](https://user-images.githubusercontent.com/14356240/29137618-e735f434-7d7b-11e7-8dbb-710a27d7cb29.jpg)
![4](https://user-images.githubusercontent.com/14356240/29137622-e77703b6-7d7b-11e7-841a-e62677c00b56.jpg)
![5](https://user-images.githubusercontent.com/14356240/29137623-e7834c48-7d7b-11e7-855b-773ce3d3cfd8.jpg)
that's all..  im lost here.. im start to use that vbox ver even it was 5fps 🤣

**Comment by cubanodun** [comment]
AkagiShiroe i have seen cases in other games that ES File explorer is detected because it come with a root explorer option 

Edit: the friend that has my account managed to roll Caster Nero twice in a single 10 roll, now im really tempted in using VBOX

**Comment by Teodor19** [comment]
Unknowns aren't problem. Yes, FGO can shit on people if it sees app that may want root.
Probably there are a lot more files than Cat had on pic.

**Comment by AkagiShiroe** [comment]
@Teodor19 if the case are like that.. it more helpfull when someone can leak it for me.. instead reading each of it > .<

**Comment by AkagiShiroe** [comment]
found the culprit libmicrovirt.so, also found a new method without deleting memu files.. rewrite all of it.

**Comment by AkagiShiroe** [comment]
ah ended failure.. can't reversed the memuguest and memuko.. need some other method to fix that.. microvirtd and microvirt-prop with other thing allready reversed.. (completely with the libmicrovirt.so). wait for good news.. or help me with the information of FGO root/emu protection.

**Comment by AkagiShiroe** [comment]
result: failure.. the only one that left are MEmuWiFi.. nothing else.. still error code 60

**Comment by Teodor19** [comment]
Are you sure that thing you doing actually hides files from FGO?

**Comment by AkagiShiroe** [comment]
@Teodor19 .. nope this time deleting the files.. not hiding anymore.

**Comment by lanzajamones** [comment]
i dont know how to do it but the point is give memu "legit" device information to bypass it.

ANRC works fine in my phone but in summer, its like a flame when playing FGO

**Comment by AkagiShiroe** [comment]
reworking on the method.. in theory should work... but its pain on the ass to repeat the method. since its temporary.. will revert back if VM/Memu restarted

**Comment by AkagiShiroe** [comment]
![test](https://user-images.githubusercontent.com/14356240/29179312-1162a492-7e2f-11e7-98a8-e4e1185bbefc.jpg)
a bit modified VM + Mapping.. done its work.. complicated... won't share it.. since im tired to write the guide.

**Comment by Teodor19** [comment]
Good job. FPS?

**Comment by RinFallen** [comment]
Someone pls give @AkagiShiroe a gold medal :v

**Comment by AkagiShiroe** [comment]
... uhm.. like usual? no impact on performance.. only mapping

**Comment by cubanodun** [comment]
@AkagiShiroe is the process to complicated? i tried VBOX and the thing threw me a BSOD i'm out of luck or my pc is to fucked up to even try 

edit: try to dual boot Remix os with another method, still no video, freaking third world machines

**Comment by AkagiShiroe** [comment]
first you have to delete MEmuguide,download(xydownload)memuime,memulauncher (install another keyboard and another launcher first) then mapping the memory to hide the core like this:
(use HxD.. string most replaced all.. also unicode string which mean.. two time each string)
the string consist this:
microvirt.enable_su
test12345.enable.12

init.svc.memud
init.svc.testd

init.svc.microvirtd
init.svc.test12345d

microvirtd.device.id
test12345d.device.id

com.microvirt.memuime
tes.test12345.testime

com.microvirt.launcher
tes.test12345.launcher

com.microvirt.download
tes.test12345.down1234

com.microvirt.guide
tes.test12345.guide

memud
testd

microvirtd
test12345d

memuguest.ko
testguest.ko

memusf.ko
testsf.ko

libmicrovirt
libtest12345

**Comment by cubanodun** [comment]
@AkagiShiroe yeah very complicated to me, time to consider join the guerrilla for a galaxy s7 ajajajaja

**Comment by AkagiShiroe** [comment]
![screenshot_2017-08-11-03-09-17](https://user-images.githubusercontent.com/14356240/29185368-f8326aec-7e43-11e7-8ab6-4f21efc70d49.png)
done.. stable enough just like old day

**Comment by cubanodun** [comment]
@AkagiShiroe good job (im jealous for your saber shiki tried to roll her for years )

**Comment by AkagiShiroe** [comment]
my friend give me this starter account after i lose my old acc which only have arthuria(lancer)np2,altera and orion

**Comment by cubanodun** [comment]
I suppose with you connection you cant upload the VM, @Teodor19 want to try and become the god emperor of emus?

**Comment by AkagiShiroe** [comment]
@cubanodun yeah if i could teach someone who could upload Modified VM( they can use old fgov2 from cat).. and also compress the method to an guide so everyone will understand it.. the rest are.. how to spread it also control it without getting busted again.

**Comment by cubanodun** [comment]
@AkagiShiroe that is why i recommended @Teodor19 he was the one uploading the modified Remix os and all the cool stuff in the reddit

**Comment by reihunter360** [comment]
keep it privateeeee

**Comment by cubanodun** [comment]
Gonna try @AkagiShiroe method, is better that this freaking blue screens of death 

Is there a way to delete Memuguide? i can only stop it in FGOv2 

and do i need root for this?

**Comment by MagicSakura1** [comment]
Is this using HxD a one time thing or will we have to constantly redo it each time we restart the app/memu?

**Comment by darkzamurai** [comment]
congratulations......

**Comment by cubanodun** [comment]
This method is very convenient even in someone has the VM they still need to mask things with HxD

edit: 1 hour trying to delete memuguide im to dumb for this

**Comment by ScienceOfMemory** [comment]
Some retard from reddit reported the fix to aniplex and bragged about it on 4chan, listing this thread as source so I suggest you delete it.

**Comment by cubanodun** [comment]
Yeah, the imbecile reported @AkagiShiroe fix to FGO USA, better to close this

**Comment by Teodor19** [comment]
So this is probably what kuutenko used for 99, if i would guess. I can't really try this, as my notebook is very dead (motherboard rotten) and this PC runs on 9800GT that doesn't have opengl support. So i can't use Memu. EDIT: WTF, it launched. Now should try lollipop. Or not, stuck on 99% instead of 59%.
As of "how to spread it also control it without getting busted again", it's too late.
http://i.4cdn.org/vg/1502407397584.png
Though it's probably a joke, as no one is stupid enough to believe in free quartz or in memu=cheat-tool.

**Comment by cubanodun** [comment]
@Teodor19 i propose a private discord, only for a few, its sad that a few edgelords that only want internet points ruin everything just for a laugh 

edit: really? can you send me a guide via PM on reddit?

**Comment by Teodor19** [comment]
Well, that depends what devs will do on next update. Will they target more files or just delete x86 folder in apk?
Hello, Aniplex guy reading this! As you can see, it's not very sophisticated method of playing, so don't touch it. A-a-and! [TALK TO US, WE ARE NOT SOME CRIMINALS!](https://www.reddit.com/r/grandorder/comments/6sqeb9/gamepress_ask_the_developers_opportunity_for_fgo/dles7lj/)

> edit: really? can you send me a guide via PM on reddit?

Hmm, what guide, the one that's above in this thread?

**Comment by cubanodun** [comment]
@Teodor19 well a most simplified version of that one if its possible, the only thing i managed to delete via console was the launcher

**Comment by Teodor19** [comment]
Well, if i manage to revive my notebook, sure. Though finding new motherboard for notebook is futile effort.

**Comment by AkagiShiroe** [comment]
@Teodor19  @cubanodun @reihunter360 never expect its spread so fast.. i delete (edit) my log up there.. hope no one spill it.. until further notice if i write the guide.

**Comment by reihunter360** [comment]
@AkagiShiroe DW saw it already, and some assholes took screenshots of it.

**Comment by AkagiShiroe** [comment]
then good luck on preventing it~.. since it was allmost impossible xD (for DW).. and also if they decide to drop x86.. then screw my life.. have to find another ARM-Based emulator.

**Comment by Teodor19** [comment]
Well, andy has arm, i think.

**Comment by AkagiShiroe** [comment]
and i state it: its no possible to cheat FGO with this memory mapping.. it will cause android reboot and ._. stupidly people won't realize it.

**Comment by AkagiShiroe** [comment]
and what kind terrible developer.. if they just do that.. instead preventing other website like *uhum.. other forum that have genuine cheat.. instead of this temporary mapping which is.. risky because sometime can force close and even android reboot screen

**Comment by AkagiShiroe** [comment]
welp, if you planning to make discord chat.. talk to me.. privately in FB, only have that.. probably group chat is better choice. im open @Teodor19 @RinFallen @reihunter360 @cubanodun .. still remember my old contact right?

**Comment by Senimaru** [comment]
SHame that because of one fking idiot other cant use this to play.... NA was a mistake.

**Comment by darkzamurai** [comment]
(Tenacious D Song) FUUUUUCK!

**Comment by RinFallen** [comment]
:sadreactonly: Some retard who tried your method but got failed then reported to DW ( i think so ) :v

**Comment by reihunter360** [comment]
Its working for me at least. >_>

**Comment by RinFallen** [comment]
@reihunter360 Haha, Then enjoy it when you still can before DW touch it.

**Comment by AkagiShiroe** [comment]
i will share it ok? just wait me to write the guide :u .. and @Teodor19 contact me, i need u to upload modified vm.

**Comment by reihunter360** [comment]
@AkagiShiroe 
dont upload modified VM
 DW will patch it. just tell them the steps. and its up to them how they can go onto doing it.
Dont spread it to the populace, only to people who actually know what they're doing.

Hell, i'm still using the FGOv2.OVA from months ago.

**Comment by AkagiShiroe** [comment]
welp any suggestion to share where? start to write the guide.

**Comment by reihunter360** [comment]
@AkagiShiroe write a guide, keep it private, only give it to people that really need it. On Facebook privately.

**Comment by jirat-sumitanun** [comment]
Pastebin or facebook chat. I think it ok.

**Comment by reihunter360** [comment]
i wrote one on my pastebin account for someone, that's all. But that guide requires a brain. :3

**Comment by AkagiShiroe** [comment]
:3 everything need a brain.. including how to hold your finger to roll gacha or trying so hard to get SSR

**Comment by AkagiShiroe** [comment]
:3 everything need a brain.. including how to hold your finger to roll gacha or trying so hard to get SSR

**Comment by wDCat** [comment]
getting SSR don't need a brain...just need money...a lot of...and a finger

**Comment by darkzamurai** [comment]
aslo a big ass to fart just kidding

**Comment by AkagiShiroe** [comment]
haha so cat you can play? i can email you if you want about the method.. and zamurai.. just contact my fb

**Comment by darkzamurai** [comment]
thanks

**Comment by darkzamurai** [comment]
wait were not friends on fb just wait for someone named barnuts dasubay and
that is me also i have screenshots from fgo.....

**Comment by lanzajamones** [comment]
i didnt have time to try it, i will beg like a worm via PM

edit: and re-rolling as a cheat? ...you make a script via Ankulua or any macro app to make a fast reroll in your phone. So blame emus for it is stupid.

**Comment by jirat-sumitanun** [comment]
@AkagiShiroe Can I have way to contact you I have a little problem?

**Comment by AkagiShiroe** [comment]
@jirat-sumitanun  post your fb link here, for some reason.. my fb are weird

**Comment by jirat-sumitanun** [comment]
@AkagiShiroe 
https://www.facebook.com/jirat.sumitnan
I just send request to join Fate/Grand Order ft. Emulators Community

**Comment by cubanodun** [comment]
oh lol i went to sleep and suddenly everything looks weird, the guide can only be obtained via facebook? :shudder:, i dont have one, i hate that thing 

nvm: i found that i made one years ago 

https://www.facebook.com/cubanodun

**Comment by ShikaBaroua** [comment]
@AkagiShiroe Is it possible to share over Facebook?
https://www.facebook.com/profile.php?id=100012699426987

**Comment by AkagiShiroe** [comment]
most likely no.. its private~

**Comment by ShikaBaroua** [comment]
No problem, the more public this is the more easily it'll get shut down. So keep it private yeah.

**Comment by kinsoru** [comment]
I've been following this thread since it was created. I tried to do the method but each time I do the first part Waifu Sim gets stuck at 99%. The second part is easy enough though. Would you be willing to clear up my confusion for the first part? If not then I understand and will just keep trying on my own. ^_^;

Edit: Recently discovered why mine would get stuck at 99%. It was because I changed permissions. After finding a different way to do the first part I'm able to load it up. After doing the second part it still doesn't work. So either the method is missing steps/parts or just doesn't work. Because I thoroughly went through the whole thing multiple times with the same result.

**Comment by lanzajamones** [comment]
I know what must be done, and yet i dont know what im doing. 
i got stuck error 70

**Comment by Teodor19** [comment]
Are you sure you're using 1.27.1 and not 1.27.0?

**Comment by falsaria** [comment]
please message me the guide in here https://www.facebook.com/valsariaeinsburg
I'll keep it private 
edit : @AkagiShiroe

**Comment by skyperion04** [comment]
pls message me the guide: https://www.facebook.com/bapbap0415. I'll keep it private.

**Comment by wDCat** [comment]
@AkagiShiroe thanks,but I'm already AFK. 
XD

**Comment by AmaterasuNyan** [comment]
Can you please send me the guide?  https://www.facebook.com/max.hobbs.735

**Comment by athenatan** [comment]
I need the guide too. thank you.
https://www.facebook.com/page.theFantasyObsessed

**Comment by ginjix** [comment]
message me the guide pls, since my phone is only 1gb ram and always crash, ty, and of course i'll keep it for myself
https://www.facebook.com/kamijou.uju

**Comment by edgyboy** [comment]
i will be very appreciated if you would send me
https://www.facebook.com/ushimushi35
thank you

**Comment by Rock13010** [comment]
Hello can you send me the guide thank you https://www.facebook.com/rachid.korifi

**Comment by Buk0595** [comment]
Can you send me a guide? https://www.facebook.com/profile.php?id=100001633980555 thanks.

**Comment by TonyJaa212** [comment]
Can you send me some nudes? Thanks.

**Comment by darkzamurai** [comment]
Read some hentai dounjins no need to ask......

**Comment by NoahAU** [comment]
I'd appreciate it if you would be willing to send me a guide as well. Thanks in case you do. 
https://www.facebook.com/lusk.noah.9

**Comment by Crayten** [comment]
Bought a tablet instead nvm~

**Comment by espamearola** [comment]
I can haz guide please? https://www.facebook.com/juan.asecas.71

**Comment by ieqa** [comment]
Not sure if you are still giving the guide but if you would send it, I really appreciate it.  Thank you~ https://www.facebook.com/reize

**Comment by lanzajamones** [comment]
i give up i have no knowledge to complete it. Keep it in private when everybody knows makes no sense but gg gl

**Comment by Kusika** [comment]
Can I get the guide too?

https://www.facebook.com/cornel.hummy

**Comment by dreamst10** [comment]
Any chance I could get the guide?
https://www.facebook.com/luisricardo.vivas

**Comment by Daranion** [comment]
Could someone send me the guide on Discord if you've received it? Daranion #6493

**Comment by Daranion** [comment]
Your guide went public with an automated patcher program by the way.

**Comment by reihunter360** [comment]
@Daranion 
What? That patcher? The one that had trojans and malware? Also the image file that immediately sent your bind code to the person spreading it?
Heh.

**Comment by AkagiShiroe** [comment]
done

**Comment by cubanodun** [comment]
Well it look it was removed, better to stay in our sikrit kool guys klub ranting about bad rolls

**Comment by reihunter360** [comment]
It was removed because it violated reddit TOS.

Come on, its a hacktool. Or a guide on how to change memory.

**Comment by AkagiShiroe** [comment]
let see DW Counter-attack if they "allready" have it.. probably on the next huge-update. like EOR3 😿 .
im counting on you @reihunter360 for dumping the shit.

**Comment by reihunter360** [comment]
@AkagiShiroe hhaa. not possible.

**Comment by AkagiShiroe** [comment]
<(") fine~ im just asking.. not forcing

**Comment by AkagiShiroe** [comment]
<(") fine~ im just asking.. not forcing

**Comment by ScienceOfMemory** [comment]
Looks like your method has some serious flaw. This went up on 4chan and looks too credible and like too much effort for a troll post: https://pastebin.com/U7qb6Jbh
TL;DR this method causes under the hood malfunctions with sending and getting data from FGO which causes DW to really notice your account as dodgy

**Comment by AkagiShiroe** [comment]
@ScienceOfMemory nope that wasn't happening, im not that fool enough yet. xD basically what we do here just renaming unnused stuff like boot configuration(that's memud,microvirtd,microvirt.sh,etc).
(except MEMU keyboard and launcher that can be replaced, and the worse case are stopped working).
and that wasn't a troll post.. just in case he/she put it to warn everyone if that was possible.. but mostly stuff like that if you doesn't use official client (which we use here and safe, apart from FGO we only modify MEmu enviroment configuration and that was totally safe in case im assure or if you tried to modify used memu system library it will end reboot)

**Comment by ScienceOfMemory** [comment]
Have you actually verified there are no hold ups with connections after doing your grand scale string replacement?

**Comment by 29988122** [comment]
To be honest the pastebin looked like legit.
You can wireshark and compare packages before and after patch took effect.
Although it claimed that it's SSl'd, but I guess there's still a way out. (either extract the key or just do simple packet size/behavior analysis)

**Comment by AkagiShiroe** [comment]
@ScienceOfMemory yeah i verified which file i replace though.. that's why i don't mention to not replacing whole "memustring".. since "memud,memuguest,memusf" are vbox-stuff(modified though to load some configuration and boot-feature), while microvirtd and other-stuff like that just contain same-stuff. i used to hack some game connection where enemies movement get stopped. (an online one).

**Comment by YellowDrago** [comment]
@AkagiShiroe Could I get the guide too if you dont mind? Had an accident with my phone and now i have no way of getting on :(. https://www.facebook.com/milo.su.7

**Comment by Dazahet** [comment]
@AkagiShiroe  if you don't mind can i get the guide too, I really want to play fate again but I can't afford a new phone
https://www.facebook.com/Dazahet

**Comment by SilverEyedCrow** [comment]
I want the guide plz >w< <3 https://www.facebook.com/SilverEyedHawk

**Comment by hoangvip80** [comment]
Can you guild me also, ill be very appreciate https://www.facebook.com/duyhoangvip
